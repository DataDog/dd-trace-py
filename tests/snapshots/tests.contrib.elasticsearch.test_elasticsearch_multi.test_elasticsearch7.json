[[
  {
    "name": "elasticsearch.query",
    "service": "elasticsearch",
    "resource": "PUT /ddtrace_index",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "elasticsearch",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "elasticsearch",
      "elasticsearch.method": "PUT",
      "elasticsearch.params": "ignore=400",
      "elasticsearch.url": "/ddtrace_index",
      "language": "python",
      "out.host": "localhost",
      "runtime-id": "39f56878a5f64e5cbe5f15a3a0c4f06e",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 40247
    },
    "duration": 50106541,
    "start": 1692709770187625755
  }],
[
  {
    "name": "elasticsearch.query",
    "service": "elasticsearch",
    "resource": "DELETE /ddtrace_index",
    "trace_id": 1,
    "span_id": 1,
    "parent_id": 0,
    "type": "elasticsearch",
    "meta": {
      "_dd.base_service": "",
      "_dd.p.dm": "-0",
      "component": "elasticsearch",
      "elasticsearch.method": "DELETE",
      "elasticsearch.params": "ignore=%5B400%2C+404%5D",
      "elasticsearch.url": "/ddtrace_index",
      "language": "python",
      "out.host": "localhost",
      "runtime-id": "39f56878a5f64e5cbe5f15a3a0c4f06e",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 40247
    },
    "duration": 9045291,
    "start": 1692709770269038130
  }]]
