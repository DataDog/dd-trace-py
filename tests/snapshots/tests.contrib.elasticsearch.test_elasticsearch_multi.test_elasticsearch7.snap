[[{"name" "elasticsearch.query"
   "service" "elasticsearch"
   "resource" "PUT /ddtrace_index"
   "type" "elasticsearch"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1615404342493231000
   "duration" 44487000
   "meta" {"runtime-id" "33c55b5f693d44d481205163652f4a4c"
           "elasticsearch.method" "PUT"
           "elasticsearch.url" "/ddtrace_index"
           "elasticsearch.params" "ignore=400"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 51898
              "_dd.tracer_kr" 1.0}}]
 [{"name" "elasticsearch.query"
   "service" "elasticsearch"
   "resource" "DELETE /ddtrace_index"
   "type" "elasticsearch"
   "error" 0
   "span_id" 0
   "trace_id" 1
   "parent_id" nil
   "start" 1615404342538001000
   "duration" 39621000
   "meta" {"runtime-id" "33c55b5f693d44d481205163652f4a4c"
           "elasticsearch.method" "DELETE"
           "elasticsearch.url" "/ddtrace_index"
           "elasticsearch.params" "ignore=%5B400%2C+404%5D"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 51898
              "_dd.tracer_kr" 1.0}}]]
